packages/gestalt/src/Image.js

```diff
@@ -81,14 +81,10 @@ export default class Image extends PureComponent<Props> {
   static defaultProps: {|
     color: string,
     fit?: 'contain' | 'cover' | 'none',
-    importance?: 'high' | 'low' | 'auto',
```

Comment:
Does the lack of a default value for `fetchpriority` matter? Should it default to "auto"?